nolearn/lasagne/visualize.py

```diff
@@ -259,3 +262,116 @@ def saliency_map_net(net, X):
 
 def plot_saliency(net, X, figsize=(9, None)):
     return _plot_heat_map(net, X, figsize, lambda net, X, n: -saliency_map_net(net, X))
+
+
+def get_hex_color(layer_type):
+    """
+    Determines the hex color for a layer.
+    :parameters:
+        - layer_type : string
+            Class name of the layer
+    :returns:
+        - color : string containing a hex color for filling block.
+    """
+    COLORS = ['#4A88B3', '#98C1DE', '#6CA2C8', '#3173A2', '#17649B', \
+          '#FFBB60', '#FFDAA9', '#FFC981', '#FCAC41', '#F29416', \
```

Comment:
Looks like you're missing four space characters at the start of this line.  Same for the following two lines.  Please remove the `\` at the end.